### PR TITLE
[MRG] Python3 compatibility

### LINKED ISSUE #128: Make code Python 3 compatible
Pypreprocess is not Python 3 compatible. It should be fairly easy to solve this automatically by fixing a couple of "print" statements,  xrange ==> range, etc.

### PR DESCRIPTION
**Pull request by lesteve**
Fix #128.

I added a build entry in Travis for Python 3 and it should pass the test on Python 3

I also made sure that `examples/easy_start/nipype_preproc_spm_auditory.py` run fine locally.

There are still some dodgy things like some `map` that I have not changed.

### CODE REVIEW

File: scripts/hcp_preproc_and_analysis.py [hunk 1]
```diff
@@ -349,8 +350,8 @@ def run_suject_level1_glm(subject_data,
 
     # fit GLM
     def tortoise(*args):
-        print args
-        print (
+        print(args)
+        print(()
```
Comment (bthirion) [inline scripts/hcp_preproc_and_analysis.py]: ?

**Comment by bthirion** [comment]
LGTM. Thanx a lot, this is super useful for the good of the project !

File: scripts/hcp_preproc_and_analysis.py (hunk 1 above)
Comment (lesteve) [inline scripts/hcp_preproc_and_analysis.py]: Yeah this is my automatic sub-standard regex replacement. Oh well I should have used futurize or something. I'll look closer at this.

**Comment by dohmatob** [comment]
Thanks plentifully, @lesteve. Merging.

**Pull request merged**